app/consistent_apphash_test.go

```diff
@@ -3,134 +3,303 @@ package app_test
 import (
 	"fmt"
 	"testing"
+	"time"
 
 	"github.com/celestiaorg/celestia-app/app"
 	"github.com/celestiaorg/celestia-app/app/encoding"
 	"github.com/celestiaorg/celestia-app/pkg/appconsts"
-	appns "github.com/celestiaorg/celestia-app/pkg/namespace"
 	"github.com/celestiaorg/celestia-app/pkg/user"
 	testutil "github.com/celestiaorg/celestia-app/test/util"
 	"github.com/celestiaorg/celestia-app/test/util/blobfactory"
+	"github.com/celestiaorg/celestia-app/test/util/testfactory"
+
```

Comment:
does your editor automatically add these newlines between imports? They seem unecessary to me. If all the imports are in one block, then golangci-lint will order them correctly.

```suggestion
```